circuitx/navigation/src/androidMain/kotlin/com/slack/circuitx/navigation/intercepting/LoggingNavigationEventListener.kt

```diff
@@ -0,0 +1,28 @@
+// Copyright (C) 2024 Slack Technologies, LLC
+// SPDX-License-Identifier: Apache-2.0
+package com.slack.circuitx.navigation.intercepting
+
+import android.util.Log
+import com.slack.circuit.runtime.screen.PopResult
+import com.slack.circuit.runtime.screen.Screen
+import kotlinx.collections.immutable.ImmutableList
+
+/** A [CircuitNavigationEventListener] that adds Logcat logging for Circuit navigation. */
+public object LoggingNavigationEventListener : CircuitNavigationEventListener {
```

Comment:
Can we hoist the impl into commonMain and then supply a logcat impl option? Similar to how Timber has a LogcatTree